tests/unit/local/docker/test_lambda_container.py

```diff
@@ -461,31 +452,7 @@ def test_must_skip_if_port_is_not_given(self):
 
 
 class TestLambdaContainer_get_image(TestCase):
-    def test_must_return_lambci_image_with_debug(self):
-        debug_options = DebugContext(debug_ports=[1235], debugger_path="a", debug_args="a=b c=d e=f")
-
-        expected = "lambci/lambda:foo"
-
-        image_builder = Mock()
-        image_builder.build.return_value = expected
-
-        self.assertEqual(
-            LambdaContainer._get_image(
-                lambda_image=image_builder,
-                runtime="foo",
-                packagetype=ZIP,
-                image=None,
-                layers=[],
-                debug_options=debug_options,
-            ),
-            expected,
-        )
-
-        image_builder.build.assert_called_with("foo", ZIP, None, [], True)
-
-    def test_must_return_lambci_image_without_debug(self):
-        debug_options = DebugContext()
-
+    def test_must_return_lambci_image(self):
```

Comment:
explain: since `build()` won't accept `debug_options` argument, these two tests can be combined as a single one.

Reply:
we could rename this test too.

Reply:
it was renamed by removing "_without_debug" in the end, what do you suggest to further rename it to?